ios/MullvadVPN/SettingsManager/SettingsManager.swift

```diff
@@ -409,6 +437,7 @@ enum SettingsKey: String, CaseIterable {
     case settings = "Settings"
     case deviceState = "DeviceState"
     case lastUsedAccount = "LastUsedAccount"
+    case isSettingsWipeCompatible = "IsSettingsWipeCompatible"
```

Comment:
Suggestion: Rename this "shouldWipeSettings"

Reply:
Agreed, much better.